styles/responsive_styles.h

```diff
@@ -59,14 +59,12 @@ using ResponsiveLightningBlockL =
   LockupTrL<
     AlphaL<COLOR,
       LayerFunctions<
-         Bump<Sin<Int<24>,Scale<BladeAngle<>,Int<14000>,Int<6000>>,Int<4000>>,
-              Sin<Int<17>,Scale<TwistAngle<>,Int<12000>,Int<9000>>,Int<8000>>>,
-         Bump<Sin<Int<27>,Scale<BladeAngle<>,Int<24000>,Int<10000>>,Int<8000>>,
-              Sin<Int<20>,Scale<TwistAngle<>,Int<10000>,Int<7000>>,Int<2000>>>,
-         Bump<Sin<Int<25>,Int<18000>,Scale<BladeAngle<24000,32768>,Int<22000>,Int<15000>>>,
-              Sin<Int<10>,Scale<TwistAngle<>,Int<6000>,Int<1000>>,Int<0>>>,
-         Bump<Sin<Int<31>,Scale<BladeAngle<22000,32768>,Int<30000>,Scale<BladeAngle<24000,32768>,Int<16000>,Int<10000>>>,Scale<BladeAngle<24000,32768>,Int<16000>,Int<8000>>>,
-              Sin<Int<18>,Scale<TwistAngle<>,Int<8000>,Int<2000>>,Int<0>>>>>, 
+         Bump<Scale<SlowNoise<Scale<BladeAngle<24000,32768>,Int<2100>,Int<1000>>>,Scale<BladeAngle<24000,32768>,Int<3000>,Int<10000>>,Int<16000>>,
```

Comment:
```suggestion
         Bump<Scale<SlowNoise<Scale<BladeAngle<24000,32768>,Int<2100>,Int<1000>>>,Scale<BladeAngle<24000,32768>,Int<3000>,Int<10000>>,Int<16000>>,
```

Reply:
These look the same to me. ALso, I've never seen these "suggested changes" before. I think you're doing it wrong.